### PR TITLE
fix(review): reconcile stale human-required blockers

### LINKED ISSUE #1648: ☂️ orchestrator: proactive board-health nudging for full autonomy
## Motivation

Sybra's board doesn't currently drive itself. Today's investigation found:

1. **No orchestrator cron is actually running.** Searched every Claude Code
   session transcript on this machine for the orchestrator's own system
   prompt ("You are the Sybra orchestrator") — zero matches. `CronList`,
   `crontab -l`, and `launchctl list` all show nothing scheduled.

2. **Even if started, the orchestrator's own bootstrapping instructions are
   stale.** `~/.sybra/CLAUDE.md` (the orchestrator brain) tells every
   session to immediately `CronCreate(schedule="*/5 * * * *",
   prompt="/sybra-monitor")`. But that skill's own file says: "**Deprecated.**
   Monitor now runs in-process inside Sybra... Do **not** register a cron or
   `/loop` for it." The brain is pointing at dead code.

3. **`orchestrator.auto_triage` / `orchestrator.auto_plan` config fields are
   vestigial.** Traced every reference in the codebase
   (`internal/config/config_orchestrator.go`,
   `internal/sybra/config_diff.go`) — outside their own struct definition,
   the only other reference is the hot-reload diff checker, which just flags
   "did this value change," never reads it to gate any behavior. Flipping
   either to `true` currently changes nothing.

4. **The real orchestrator-start gate is narrower than it looks.**
   `maybeStartOrchestrator` (`internal/sybra/app_orchestrator.go`) only
   auto-starts the orchestrator session when a task is already
   `planning`/`plan-review`/`in-progress`/`in-review` — it never checks
   `todo`. So the orchestrator is only ever a byproduct of something else
   already being in flight, never the thing that proactively starts new work
   or re-checks stuck work.

5. **Nothing proactively reconciles resolved blockers or nudges stalled
   tasks.** The in-process Go monitor (`internal/monitor/`) is real and
   active — it reacts to specific anomaly signatures (lost_agent,
   stuck_human_blocked, PR gaps) — but nothing periodically re-checks a
   `human-required`/`blocked` task whose original blocking reason may have
   since cleared, or distinguishes a genuinely stuck task from normal churn
   and takes a corrective action (retry dispatch, resync a status/workflow
   desync, force-push a diverged branch). See #1641 for one concrete
   instance of this gap (a DCO-blocked task sat `human-required` for hours
   after DCO actually passed, with nothing to notice and reconcile it).

Today, all of this was done manually and reactively — investigating stuck
tasks, diagnosing root causes, unblocking, filing issues — a full session's
worth of exactly the job the orchestrator is supposed to do autonomously.

## Goal

Sybra's board should proactively drive itself: nudge tasks that have
stopped progressing, reconcile blockers whose underlying condition has
cleared, and only surface genuinely judgment-requiring work to a human.

### 1. Deterministic reconciliation (in-process Go, cheap, no LLM cost)

Extend `internal/monitor/` to periodically re-probe `human-required`/
`blocked` tasks whose `statusReason` names a specific, machine-checkable
condition (a named CI check, DCO status, a merged PR/issue reference) and
reconcile the status automatically when that condition now holds. #1641 is
the first concrete case of this.

### 2. Periodic judgment-driven nudging (recurring scheduled agent pass)

A recurring pass (cloud `/schedule`, not a session-local loop — this needs
to survive across sessions) that scans the board for tasks that have
stopped progressing and applies the same judgment a human/session would:
distinguish a genuine stall from normal churn, decide whether a diverged
branch needs syncing, spot a status/workflow desync, recognize a
false-positive test failure or misdiagnosed escalation, and take the
corrective action — or leave it alone and explain why when it's a genuine
judgment call for a human.

### Prerequisite cleanup

- Fix `~/.sybra/CLAUDE.md`'s orchestrator brain: stop instructing sessions
  to register a cron for the deprecated `/sybra-monitor` skill; describe the
  actual current duties instead.
- Remove or properly wire `orchestrator.auto_triage`/`auto_plan` — currently
  dead config that implies control it doesn't have.
- Consider whether `maybeStartOrchestrator` should also trigger on a
  `todo`-only backlog, not just when something is already active.

## Related

- #1641 — human-required tasks never reconcile on cleared blocker (one
  concrete instance of the deterministic-reconciliation gap this issue
  generalizes).

### PR DESCRIPTION
**Pull request by Automaat**
## Motivation

Tasks parked in `human-required` after the pr-monitor auto-fix retry budget
was exhausted (CI failure or merge conflict) could stay stuck even after the
underlying PR became mergeable again — nothing re-probed the live PR state
to notice the blocker had cleared on its own.

## Implementation information

- Added `humanRequiredBlockerReconcilable`/`reconcileHumanRequiredBlockers` in
  `internal/sybra/review/outbound.go`: re-probes the linked PR of eligible
  `human-required` tasks (parked solely by a `ci_failure`/`conflict`
  auto-fix-exhausted escalation) and moves them back to `in-review` once the
  PR is open, mergeable, and CI is green with nothing pending.
- Excludes watchdog stops, tamper-flagged reasons, comment-review exhaustion,
  and tasks already reconciled once (`monitor:reconciled` latch tag) to avoid
  flip-flopping.
- Follow-up commit narrows `humanRequiredBlockerReconcileEligible` back to
  reporting reconciliation eligibility generically, per review feedback.
- Related config/audit/frontend cleanup around the orchestrator settings
  panel and audit event plumbing.

Closes https://github.com/Automaat/sybra/issues/1648

_Generated by Sybra harness_

### CODE REVIEW

File: internal/sybra/review/outbound.go [hunk 1]
```diff
@@ -241,55 +266,82 @@ func hasFixableIssue(issues []github.PRIssue) bool {
...
-			{ID: t.ID, PRNumber: t.PRNumber, Branch: t.Branch, ProjectID: t.ProjectID},
-		})
-		if hasFixableIssue(issues) {
-			continue
+
+		probe := map[string]any{}
+		if pr := matchingPR(t, byNumber, byBranch); pr != nil {
+			if pr.IsDraft || pr.CIStatus == "PENDING" || pr.HasPendingChecks {
+				continue
+			}
```
[the remark this answers is not in the document]
Comment (Automaat) [reply internal/sybra/review/outbound.go:294]: Tightened the monitored-PR fast-path to use the same concrete readiness gate as the fallback probe: `Mergeable == "MERGEABLE"`, `CIStatus == "SUCCESS"`, no pending checks, and not draft. I also added focused tests for the two cases that were slipping through the snapshot path: `Mergeable == "UNKNOWN"` and empty CI.

_Generated by Sybra harness_
